Makefile

```diff
@@ -1,5 +1,5 @@
 format:
-	stylua -v --verify lua/rocks/ plugin/
+    stylua -v --verify lua/rocks/ plugin/
```

Comment:
seems like a mistake

Reply:
I think that was editorconfig complaining

Reply:
I mean dont makefiles only accept tab before commands ?

Reply:
You're probably right ..